hoomd/md/TwoStepConstantVolumeGPU.cu

```diff
@@ -77,7 +77,7 @@ __global__ void gpu_nvt_rescale_step_one_kernel(Scalar4* d_pos,
         int3 image = d_image[idx];
 
         // time to fix the periodic boundary conditions
-        box.wrap(pos, image);
+        box.wrap(pos, vel, image);
```

Comment:
Please confirm anywhere you're just using the existing velocity variable that you've been given that it is (1) not const and (2) its array handle was not read only. The compiler should yell about the first one but not the second.